Add alpha2 code to user language

s-petit opened the pull request:
Add alpha2 code to user language
    
    This commit is backwards compatibl. We just add an ISO639-1  alpha2code. The default code is ISO 639-2 alpha-3.
    
    Depending on projects, we may need one or another. It will be easier to use the most pertinent code depending on context

s-petit committed:
Add alpha2 code to user language

This commit is backwards compatible. We just add an ISO639-1  alpha2code. The default code is ISO 639-2 alpha-3.

Depending on projects, we may need one or another. It will be easier to use the most pertinent code depending on context

fchetouani commented:
Une petite coquille dans le message de commit :-)
compatibl deviendrait compatibl**e**

fchetouani reviewed (approved):
:+1:

ohue-ruth reviewed (approved):
C'est bon avec tes explications et ceux de @fchetouani, j'ai refait le cheminement entre sesame/babel pour avoir le code alpha 2 et c'est entièrement limpide. Merci :relaxed: !

s-petit committed:
Allow minor release with npm run minor-release

We only had a script to make major releases.

We also need to make minor releases, when we add backward compatible features.

According to semver, only breaking features should lead to a major release.

s-petit committed:
14.1.0
